dbt/include/trino/macros/materializations/materialized_view.sql

```diff
@@ -1,5 +1,9 @@
 {%- macro trino__get_create_materialized_view_as_sql(target_relation, sql) -%}
   create materialized view {{ target_relation }}
+  {%- set grace_period = config.get('grace_period') -%}
+  {%- if grace_period is not none -%}
```

Comment:
```suggestion
  {%- if grace_period is not none %}
```
Please do not strip whitespace between table name and `grace period` clause